tests/integration/kubernetes/gha-run.sh

```diff
@@ -450,6 +450,8 @@ function cleanup() {
 		return
 	fi
 
+	# Delete all test scripts
```

Comment:
Let's expand this comment to explain *why* we're doing this and reference #9980.

Reply:
Done.